run-test.sh

```diff
@@ -336,21 +354,9 @@ do
         --corefx-native-bins)
         CoreFxNativeBins=$2
         ;;
-		# *** start WCF Content ***
-        --wcf-tests)
-        WcfTests=$2
-        ;;
-        --wcf-bins)
-        WcfBins=$2
-        ;;
-        --bridge-host)
-        BridgeHost=$2
-        export BridgeHost
+        --corefx-packages)
+        CoreFxPackages=$2
         ;;
-        --xunit-args)
-        XunitArgs=$2
-        ;;
-		# *** end WCF Content ***
```

Comment:
I think we should retain this part, just in case we need to use it again.  But because we don't know where CoreFx is taking this script, it's hard to say.  Not blocking, but if you remove this part, we should probably also remove the echos at the top of the file that describe these parameters.